api/app/controllers/spree/api/shipments_controller.rb

```diff
@@ -127,7 +127,8 @@ def transfer_to_shipment
           current_shipment: @original_shipment,
           desired_shipment: @desired_shipment,
           variant: @variant,
-          quantity: @quantity
+          quantity: @quantity,
+          track_inventory: Spree::Config.track_inventory_levels
```

Comment:
I'm thinking this new param is not necessary since `Spree::Config.track_inventory_levels` could be accessed directly within `FulfilmentChanger`, but given we need to deprecate the previous behavior I can't think of any other way than this 🤔 . I'm wondering if we should try to remove it later, when we finally remove the default behavior.

Reply:
My idea was that at the end it's not that bad because this `track_inventory` value determines heavily the logic used in the class, and it's better to be as explicit as possible about it when it's initialized. After the deprecation period, we could evaluate removing it (with another deprecation), though.